src/App.tsx

```diff
@@ -1,8 +1,24 @@
 import React from "react";
+import { ApolloClient, InMemoryCache, ApolloProvider } from "@apollo/client";
 import "./styles.scss";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+
+import CharacterDetails from "./pages/CharacterDetails";
```

Comment:
new line

Reply:
I intentionally add a new line to separate components, hooks, styles, and icons.

Reply:
I meant add a new line below to seperate the component and import, there isn't any new line